sigs.yaml

```
@@ -2092,13 +2092,13 @@ sigs:
       company: Lyft
   meetings:
   - description: Regular SIG Meeting
-    day: TBD
-    time: TBD
+    day: Tuesday
+    time: 2 PM
```

silly style nit, everything else in sigs.yaml uses this format, I'd rather keep us consistent:
```suggestion
    time: "14:00"
```

---

done :)